besu/src/test/java/org/hyperledger/besu/cli/subcommands/storage/TrieLogHelperTest.java

```diff
@@ -94,10 +93,11 @@ public static void setup() throws IOException {
         .getTrieLogStorageTransaction()
         .put(blockHeader5.getHash().toArrayUnsafe(), Bytes.fromHexString("0x05").toArrayUnsafe());
     updater.getTrieLogStorageTransaction().commit();
+
+    createDirectory();
```

Comment:
Could use `@TempDir`?

Reply:
It is using, but I forgot to remove this form the setup. thanks